### ISSUE TITLE
Accessibility issue with lgPage component

### ISSUE THREAD

**Issue by NemirStephan** [Legal-and-General/canopy#427]
I am getting a 'All page content should be contained by landmarks` error in axe dev tools when using the lgPage component

Element location
```
.lg-page__skip-link
```
Element source
```
<a class="lg-page__skip-link"> Skip to main content </a>
```
I think the issue is because the `a` tag is not wrapped inside the main content area which contains the `main` aria role 
<img width="506" alt="Screenshot 2021-06-22 at 17 01 08" src="https://user-images.githubusercontent.com/22469863/122961191-106e4680-d37c-11eb-90ba-74fd6bf70075.png">

The fix should be to either declare the `main` role at the component root, or add another role such as `banner` to the anchor tag

**Comment by elenagarrone** [comment]
Yes, I agree. Ideally it should be in the header before the navigation.

**Comment by pauleustice** [comment]
@Legal-and-General/canopy-regular-contributors This sounds like a legit issue that should be quite easy to fix. I'm not sure the best way to place it in the lg-header as per Elena's suggestion, though.

Will the header component always precede a `[role=main]`? Maybe it should be an input to toggle whether it's shown in the header?
